server/chainquery/queries/claimQueries.js

```diff
@@ -70,15 +70,25 @@ export default (db, table, sequelize) => ({
 
   getAllChannelClaims: async (channelClaimId, bidState) => {
     logger.debug(`claim.getAllChannelClaims for ${channelClaimId}`);
-    const whereClause = bidState || {
+    const whereBidClause = bidState || {
       [sequelize.Op.or]: [
         { bid_state: 'Controlling' },
         { bid_state: 'Active' },
         { bid_state: 'Accepted' },
       ],
     };
+    const whereContentClause = {
```

Comment:
This needs to be optional, chainquery functions should not be opinionated on Spee.ch content types.